mappings/net/minecraft/util/math/Range.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_6497 net/minecraft/util/math/Range
```

Comment:
Potential conflict with #2689

Reply:
I personally prefer my own mappings for that class, but I'll remove them to not get conflicts.